src/data/i18n/default/transifex_input.json

```diff
@@ -50,8 +50,8 @@
   "assetsPageNoResultsMessage": "No files were found.",
   "assetsPageNoAssetsNumFiles": "0 files in your course",
   "assetsPageNoAssetsMessage": "Enhance your course content by uploading files such as images and documents.",
-  "assetsResultsCountFiltered": "Showing {start}-{end} out of {total} possible matches.",
-  "assetsResultsCountTotal": "Showing {start}-{end} out of {total} total files.",
+  "assetsResultsCountFiltered": "Showing {start}-{end} out of {total, plural, one {1 possible match} other {{total} possible matches}}.",
+  "assetsResultsCountTotal": "Showing {start}-{end} out of {total, plural, one {1 total file} other {{total} total files}}.",
```

Comment:
I think this format matches [Transifex specification for plurals](https://docs.transifex.com/formats/json#plurals-support)